modules/troubleshooting-enabling-kdump-day-one.adoc

```diff
@@ -68,6 +68,19 @@ systemd:
 <4> If you want to change the contents of `/etc/sysconfig/kdump` from the default, include this section and modify the `inline` subsection accordingly.
 <5> For the `ppc64le` platform, replace `nr_cpus=1` with `maxcpus=1`, which is not supported on this platform.
 
+[NOTE]
+====
+To export the dumps to NFS targets, the `nfs` kernel module must be explicitly added to the configuration file:
++
```

Comment:
this `+` doesn't seem to be rendering correctly in the preview...i guess being in the admonition is making it behave weird. since we don't need to chain it to a particular step, it won't hurt anything if it's removed, so let's go ahead and do that. @jbtrystram could you remove it?